### PR TITLE
Move to ECR for docker images

### PR DESCRIPTION
**Pull request by DeNeutoy**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM allenai-docker-private-docker.bintray.io/cuda:8
+FROM 896129387501.dkr.ecr.us-west-2.amazonaws.com/infrastructure/aristo/cuda:8
```
Comment (matt-gardner) [inline Dockerfile]: Have you checked with aristo-eng or techops that this name is good to put in the repo?

Comment (matt-gardner) [inline Dockerfile]: I'm guessing it's fine, but we should probably check, just to be sure.

File: scripts/run_on_aws.sh [hunk 2]
```diff
@@ -13,10 +13,22 @@ fi
 
 set -e
 
-docker pull allenai-docker-private-docker.bintray.io/cuda:8
-
-docker build -t allenai-docker-private-docker.bintray.io/$CONTAINER_NAME . --build-arg PARAM_FILE=$PARAM_FILE
+# Get temporary ecr login.
+eval $(aws --region=us-west-2 ecr get-login)
+
+docker pull 896129387501.dkr.ecr.us-west-2.amazonaws.com/infrastructure/aristo/cuda:8
```
Comment (matt-gardner) [inline scripts/run_on_aws.sh]: This long crazy string is repeated a bunch of times; can you put it in a variable up above?

**Comment by matt-gardner** [comment]
Fixes #303.

**Comment by DeNeutoy** [comment]
I'm getting an error when running the actual script which isn't due to this change - I think it was to do with removing the paths when I moved run_model into deep_qa:
```
Traceback (most recent call last):
  File "scripts/run_model.py", line 4, in <module>
    from deep_qa import run_model
ImportError: No module named 'deep_qa'

```

**Comment by matt-gardner** [comment]
Yeah, I'm about to fix that in another PR, that also adds a bit of functionality.  I'll submit it now.

File: Dockerfile (hunk 1 above)
Comment (DeNeutoy) [inline Dockerfile]: Yep it's fine.

**Pull request merged**